python/ray/services.py

```diff
@@ -77,7 +77,10 @@
     os.path.abspath(os.path.dirname(__file__)),
     "core/src/credis/build/src/libmember.so")
 
-# Location of the raylet executable.
+# Location of the raylet executables.
+RAYLET_MONITOR_EXECUTABLE = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)),
+    "core/src/ray/raylet/raylet_monitor")
```

Comment:
I believe the executable also needs to be added to the `ray_files = [` list in `python/setup.py`.

Reply:
Thanks!